src/lib/services.js

```diff
@@ -1,6 +1,21 @@
 import { APP_SLUG } from 'src/constants'
 import { JOBS_DOCTYPE } from 'src/doctypes'
 
+import { initTranslation } from 'cozy-ui/transpiled/react/providers/I18n'
+
+/**
+ * Init polyglot and returns t
+ * @returns {object}
+ */
+export const initPolyglot = () => {
+  const lang = process.env.COZY_LOCALE || 'en'
+  const dictRequire = lang => require(`locales/${lang}`)
+  const polyglot = initTranslation(lang, dictRequire)
+  const t = polyglot.t.bind(polyglot)
+
+  return { t }
+}
```

Comment:
That looks like something that should be in a separate lib? Did you take it somewhere?

Reply:
non c'est mes doigts qui ont tapé tout ça :) Effectivement on pourrait mutualiser à l'avenir...